project1/index.html

```diff
@@ -0,0 +1,142 @@
+                <span><strong><em>I</em></strong></span> was."</p>
+            <!--I used a span around the 'I' because I wanted to add color to just the 'I' to add more emphasis because I don't think it was as noticeable that I was emphasizing it with just the <em> tag-->
+        </blockquote>
+
+        <div>
+            &bull; &bull; &bull;
+            <!--not sure if this is correct, but I wanted to put these as a separator of the sections and I am having trouble
+        making the spacing above and below to be equal-->
+        </div>
+
+        <section>
+            <p> I went to Texas to study the whooping crane because I was researching a novel. In my novel there were
+                biologists doing field research about birds and I had no idea what field research actually looked like
+                and
+                so the
+                scientists in my novel draft did things like shuffle around great stacks of papers and frown. The good
+                people of the Earthwatch organization assured me I was welcome on the trip and would get to participate
+                in
+                “real
+                science” during my time on the gulf. But as I waited to be picked up by my team in Corpus Christi, I was
+                nervous—I imagined everyone else would be a scientist or a birder and have daunting binoculars.</p>
+
+            <!--I don't like that it separated my text like this is there a way to fix it or clean it up?-->
+
+            <p>The biologist running the trip rolled up in in a large white van with a boat hitch and the words
+                <strong><em>BIOLOGICAL SCIENCES</em></strong> stenciled across the side. Jeff was forty-ish, and wore
+                sunglasses and a backward baseball cap. He had a winter beard and a neon-green cast on his left arm.
+                He’d
+                broken his
+                arm playing hockey with his sons a week before. The first thing Jeff said was, <i>“We’ll head back to
+                    camp,
+                    but I hope you don’t mind we run by the liquor store
+                    first.”</i> I felt more optimistic about my suitability for science.</p>
+        </section>
+
+
+        <div>
+            &bull; &bull; &bull;
+            <!--not sure if this is correct, but I wanted to put these as a separator of the sections and I am having trouble
+        making the spacing above and below to be equal-->
+        </div>
+
+        <section class="last">
+            <!--Here, I was trying to figure out how ti make the bottom margin of the last sentence match the margin in h1-->
+            <p>Not long before I’d called off my engagement it was Christmas.</p>
+
+            <!--Here also, basically all over, I've been trying to clean it up myself, but it doesn't really appear to help much!-->
+
+            <p>The woman who was <i>supposed</i> to be my mother-in-law was a wildly talented quilter and made stockings
```

Comment:
Here, `em` is appropriate because it should be emphasized:
```suggestion
            <p>The woman who was <em>supposed</em> to be my mother-in-law was a wildly talented quilter and made stockings
```